completed.js

```diff
@@ -0,0 +1,12 @@
+const tasks = JSON.parse(localStorage.getItem('localTasks')) ?? [];
+export default class Completed {
+  static add = (id) => {
+    // const tasks = JSON.parse(localStorage.getItem('localTasks')) ?? [];
```

Comment:
I think it is better if you remove unused code comments. This way you can make your code clean.